include/pybind11/common.h

```diff
@@ -66,6 +58,13 @@
 #pragma warning(pop)
 #endif
 
+#include <vector>
```

Comment:
When compiling the examples against python3.2, there were many warnings of redefined macros.  Based on [this stackoverflow post](http://stackoverflow.com/questions/20300201/why-python-h-of-python-3-2-must-be-included-as-first-together-with-qt4) and [this documentation](https://docs.python.org/3/extending/extending.html), `Python.h` must be included before inclusion of the standard headers as it may redefine some standard library macros.  This occurred only when using python3.2 version, and not when using python2.7 or python3.4, but the warning in the documentation is present for all versions.